snap/runnable.go

```diff
@@ -0,0 +1,39 @@
+package snap
+
+import "fmt"
+
+// Runnable represents a runnable element of a snap. This could either be an
+// app, a hook, or a component hook.
+type Runnable struct {
+	// CommandName is the name of the command that is run when this runnable
+	// runs.
+	CommandName string
+	// SecurityTag is the security tag associated with the runnable. Security
+	// tags are used by various security subsystems as "profile names" and
+	// sometimes also as a part of the file name.
+	SecurityTag string
+}
+
+// AppRunnable returns a Runnable for the given app.
+func AppRunnable(app *AppInfo) Runnable {
```

Comment:
can't this be a method on AppInfo?

Reply:
Done in 3a440be92457811239c58724d791dac7d4adc5f6.